support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc

```diff
@@ -9,7 +9,7 @@ endif::[]
 
 toc::[]
 
-Insights repeatedly analyzes the data Insights Operator sends. Users of {product-title} can display the report in the *Insights* tab of each cluster on {cluster-manager-url}.
+Insights repeatedly analyzes the data Insights Operator sends. Users of {product-title} can display the report in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].
```

Comment:
The same applies to this line, you could consider replacing "link:[https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console]" with "{cluster-manager-url}" for consistency with this docs library.

Reply:
see above